spec/openapi.yaml

```diff
@@ -720,6 +768,7 @@ components:
           description: The duration of the activity, in seconds.
         submittedDate:
           type: long
+          deprecated: true
```

Comment:
@pfilippi24 this is being deprecated in favour of `submissionTime` on the time group.